external/contracts/set/BasicIssuanceModule.sol

```diff
@@ -132,11 +143,16 @@ contract BasicIssuanceModule is ModuleBase, ReentrancyGuard {
         address _to
     )
         external
+        override
         nonReentrant
         onlyValidAndInitializedSet(_setToken)
     {
         require(_quantity > 0, "Redeem quantity must be > 0");
 
+        address hookContract = _callManagerPreRedeemHooks(_setToken, _quantity, msg.sender, _to);
```

Comment:
DIM doesn't have manager pre redeem hooks. not sure if we need that even if we add module hooks.

Reply:
Yeah, since neither DIM nor BIMv1 have this, I guess it wouldn't be needed. The only example of managerHooks I found in our tests is the max supply hook (see [here](https://github.com/bergben/index-coop-smart-contracts/blob/master/test/hooks/supplyCapIssuanceHook.spec.ts)) which only needs to be called at issuance.